scripts/sanitycheck

```diff
@@ -316,6 +317,86 @@ class Handler:
         self.lock.release()
         return ret
 
+
+class DeviceHandler(Handler):
+
+    def __init__(self, instance):
+        """Constructor
+
+        @param instance Test Instance
+        """
+        super().__init__(instance)
+
+        self.instance = instance
+        self.timeout = instance.test.timeout
+        self.sourcedir = instance.test.code_location
+        self.outdir = instance.outdir
+        self.run_log = os.path.join(self.outdir, "run.log")
```

Comment:
I don't see this being used anywhere. How is it different from handler_log?

Reply:
right, was a copy from another handler, need to refactor the handler class better and stop redeclaring this in subclasses. I have this refactoring on the TODO.